.github/workflows/tests.yaml

```diff
@@ -225,4 +227,4 @@ jobs:
         run: |
           . venv\\Scripts\\activate
           pip list | grep 'astroid\|pylint'
-          python -m pytest --durations=10 --benchmark-disable tests/
+          python -m pytest --durations=10 --benchmark-disable tests/
```

Comment:
```suggestion
          python -m pytest --durations=10 --benchmark-disable tests/
```

Reply:
> This suggestion is invalid because no changes were made to the code.

Reply:
I think this is the line endings thing; I'll fix both in one commit when I get home.